docs/websockets.rst

```diff
@@ -79,19 +79,50 @@ The messages sent by rotki when a user is logging in and a db upgrade is happeni
 ::
 
     {
-        "start_db_version": 26,
-        "target_db_version": 35,
-        "current_upgrade": {
-            "from_db_version": 30,
-            "total_steps": 8,
-            "current_step": 5
-        }
+        "type": "db_upgrade_status",
+	"data": {
+	    "start_version": 26,
```

Comment:
notice the changes here for all variable names.

Also notice that in the `current_upgrade` we are now tracking the actual upgrade version as `to_version` instead of the from_db_version.